WS2012R2/lisa/remote-scripts/ica/perf_ntttcp_client.sh

```diff
@@ -510,28 +510,135 @@ if [ -d  $HOME/$log_folder ]; then
     rm -rf $HOME/$log_folder
 fi
 
-mkdir $HOME/$log_folder
-# Determine test type lagscope/ntttcp
-if [[ "${TEST_TYPE:-UNDEFINED}" = "UNDEFINED" ]] || [[ "${TEST_TYPE}" = "ntttcp" ]]; then
+mkdir  "$HOME/$log_folder"
+# Determine test type lagscope/ntttcp/ntttcp-udp
+if [[ "${UDP_MODE}" = "yes" ]]; then
     eth_log="$HOME/$log_folder/eth_report.log"
-    echo "#test_connections    throughput_gbps    average_packet_size" > $eth_log
+    echo "#test_connections    throughput_gbps    average_packet_size" > "$eth_log"
     sleep 10
     #Starting test
-    cat /proc/interrupts > $HOME/$log_folder/sender-interrupts-start.log
-    ssh -i $HOME/.ssh/${SSH_PRIVATE_KEY} -o StrictHostKeyChecking=no ${SERVER_OS_USERNAME}@${STATIC_IP2} "cat /proc/interrupts > $HOME/$log_folder/receiver-interrupts-start.log"
-    previous_tx_bytes=$(get_tx_bytes $ETH_NAME)
-    previous_tx_pkts=$(get_tx_pkts $ETH_NAME)
+    cat /proc/interrupts > "$HOME/$log_folder/sender-interrupts-start.log"
+    ssh -i "${HOME}/.ssh/${SSH_PRIVATE_KEY}" -o StrictHostKeyChecking=no ${SERVER_OS_USERNAME}@${STATIC_IP2} \
+    "cat /proc/interrupts > $HOME/$log_folder/receiver-interrupts-start.log"
+    previous_tx_bytes="$(get_tx_bytes "$ETH_NAME")"
+    previous_tx_pkts="$(get_tx_pkts "$ETH_NAME")"
     port_change=false
     port=
     for current_test_threads in "${TEST_THREADS[@]}"
     do
-        if [ $current_test_threads -lt $MAX_THREADS ]
+        if [ "$current_test_threads" -lt "$MAX_THREADS" ]
+        then
```

Comment:
can you use one type of 'if statement'? I mean in some locations is 'if cmd ;; then' and in other is
'if cmd
then'